librz/core/cmd_analysis.c

```diff
@@ -8195,7 +8195,8 @@ static int cmd_analysis_all(RzCore *core, const char *input) {
 	case 'o': // "aao"
 		cmd_analysis_objc(core, false);
 		break;
-	case 'e': // "aae"
+	case 'e': { // "aae"
+		bool reg_flags_defined = !!rz_flag_space_count(core->flags, RZ_FLAGS_FS_REGISTERS);
```

Comment:
i do not think the double negation is really needed. the definition of bool is int and false is 0 while true is everything else.

Reply:
ok 51bf92e